### PR TITLE
[13.0][MIG] base_substate

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration

### CODE REVIEW

File: base_substate/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Akretion (<http://www.akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Base Sub State",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "author": "Akretion, " "Odoo Community Association (OCA)",
```
Comment (Saran440) [inline base_substate/__manifest__.py:8]:
```suggestion
    "author": "Akretion, Odoo Community Association (OCA)",
```

File: base_substate/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "website": "https://github.com/OCA/sale-workflow/",
```
Comment (Saran440) [inline base_substate/__manifest__.py:9]:
```suggestion
    "website": "https://github.com/OCA/server-ux",
```

**Review by Saran440 — COMMENTED** [review]
Code Reviewed.
No major changed.

**Comment by kittiu** [comment]
@mourad-ehm 
Can you review? I put the difference with v12 here, https://github.com/OCA/server-ux/pull/199/commits/c0369d27cac5f91b4e6d2c15dc43d8d858eb0986
I will merge commit before merge.

**Review by ps-tubtim — APPROVED** [review]
Functional Test: OK. Code: OK.

**Comment by ps-tubtim** [comment]
@kittiu Please check Travis.

**Comment by kittiu** [comment]
@mourad-ehm can you help review this. Thanks.

**Review by mourad-ehm — APPROVED** [review]

**Pull request merged**